Triangulation_3/include/CGAL/Triangulation_3.h

```diff
@@ -4226,19 +4232,21 @@ make_canonical(Vertex_triple& t) const
   Vertex_handle tmp;
   switch(i)
   {
-    case 0: return;
+    case 0: break;
```

Comment:
I suggested to rename that function `make_canonical` to a name less confusing.

Reply:
I suggest `make_canonical_wrt_orientation()`, and to add a comment.
Do you think this name is ok?

Reply:
`make_canonical_oriented_triplet`?

Reply:
Anything with the notion of "orientation" in the name will be good.

Reply:
I chose `make_canonical_oriented_triple` to be consistent with `make_vertex_triple` in the same file

Reply:
Note that like we have `std::pair<T,T> make_sorted_pair(T t1, T t2)`,we could have `tuple<T...> make_sorted_tuple(T... t)` (since it seems you are always building the tuple before calling the function).

Reply:
> Note that like we have `std::pair<T,T> make_sorted_pair(T t1, T t2)`,we could have `tuple<T...> make_sorted_tuple(T... t)` (since it seems you are always building the tuple before calling the function).

Actually, here, the tuple is not really sorted: it is rotated, so that its first element is the smallest. That preserves the orientation of the triangle that the triple of vertices represents.